docs/technical-doc/Endorsements.rst

```diff
@@ -0,0 +1,118 @@
+============
+Endorsements
+============
+
+Intro
+=====
+
+
+Massa uses the Proof-of-Stake selection mechanism with Nakamoto consensus.
+In that context, when there are multiple cliques in close competition,
+we want all nodes to converge towards a single clique as fast as possible to minimize finality time and maximize the quality of the consensus.
+To achieve this, we draw inspiration from Tezos and introduce the concept of Endorsement.
+
+Basic principle
+===============
+
+At every slot `S`, we use the existing Proof-of-Stake selection mechanism to not only draw the block creator for that slot,
+but also `E` other stakers that are responsible for creating Endorsements that can be included in the blocks of slot `S`.
+Every endorsement that is selected to be included in blocks of slot `S` is signed by its creator `C`
```

Comment:
I would split that sentence.

A block includes some endorsements. An endorsement contains the "endorsed" hash of the parent block. C, the creator of the endorsements, would choose that parent in the same thread of the slot 'S'. As well the endorsement's creators vote for the parent of the block in their own thread.

Every endorsement that is selected to be included in the Block of slot 'S' => Do you mean "Every endorsements in the block of slot S" ?

I guess the second sentence "Blocks in slot `S` can only include endorsements that endorse the block's parent block in the thread of `S`." is redundant, or can be resumed.

Reply:
rewrote the paragrpah, hopefully more clearly